src/lib/ParseApp.js

```diff
@@ -404,13 +406,11 @@ export default class ParseApp {
   }
 
   isLocalizationAvailable() {
-    let path = '/apps/' + this.slug + '/is_localization_available';
-    return AJAX.abortableGet(path);
+    return this.serverInfo.features.push.localization === true;
```

Comment:
as written, this is exactly the same as just ` return this.serverInfo.features.push.localization` it's like boolean day or something....

if `this.serverInfo.features.push.localization` might just be truthy instead of actual `true` then `===` wont work and you'd want `return !!this.serverInfo.features.push.localization` instead?

Reply:
you are right :)

Reply:
https://www.urbandictionary.com/define.php?term=Even%20a%20broken%20clock%20is%20right%20twice%20a%20day